src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -1771,11 +1771,12 @@ def forward(
                 decoder_attention_mask=decoder_attention_mask,
                 output_attentions=output_attentions,
                 output_hidden_states=output_hidden_states,
-                return_dict=return_dict,
+                return_dict=True,  # toggle for easier access to loss/logits below
```

Comment:
Sorry 😐 realized this would break torch.script or fx export compatibility so maybe False by default ? (I might be wrong tho, but I don't think it's suported)

Reply:
yeah, torchscript is not supported for BLIP afaik and the tests are disabled therefore. I guess in that case we don't need it to be `False`

Reply:
No but you could script only the LM model and not the full model no?

Reply:
I added torchscript tests and they are passing currently. FX test cannot be added because the model architecture is not in supported list

I don't think we should do `False` be default, as that would add more complexity than before when we passed the actual `return_dict`. We'd have to wrap outputs from tuple into the correct `ModelOutputClass` manually if `return_dict`. If you think we should still not set `True` by default let's get to the very first solution I proposed

Reply:
Okay sounds good!